twitter-connector-demo/pom.xml

```diff
@@ -18,10 +19,10 @@
       <plugin>
         <groupId>com.axonivy.ivy.ci</groupId>
         <artifactId>project-build-plugin</artifactId>
-        <version>9.4.1</version>
+        <version>10.0.0</version>
         <extensions>true</extensions>
         <configuration>
-          <ivyVersion>9.4.0</ivyVersion>
+          <ivyVersion>10.0.0</ivyVersion>
```

Comment:
@alexsuter or better use `[9.4.0,]`?

Reply:
this should not be needed, please remove this property.